txircd/user.py

```diff
@@ -844,6 +835,17 @@ def modeString(self, toUser):
 			return "{} {}".format("".join(modeStr), " ".join(params))
 		return "".join(modeStr)
 
+	def resolveDNS(self, ip):
```

Comment:
I thought of that after I submitted the PR and you are most definitely right. Will fix later.